array-methods/for-each.js

```diff
@@ -0,0 +1,12 @@
+const values = [10, 1, 22, 23, 41, 5, 18, 7, 80, 9];
+const orderedValues = [];
+values.forEach((value) => {
+  orderedValues.push(value);
+});
+console.log('orderedValues:', orderedValues);
```

Comment:
You should be logging each value, not logging the whole array. You also shouldn't be creating a new array